Demo/src/main/java/com/braintreepayments/demo/GooglePayFragment.java

```diff
@@ -57,27 +53,13 @@ public View onCreateView(@NonNull LayoutInflater inflater,
     public void onResume() {
         super.onResume();
 
-        braintreeClient.getConfiguration((configuration, error) -> {
```

Comment:
Curious - why were we doing config fetches in the demo app?

Reply:
Yeah we probably could've deleted this a while back. This has [roots in v3](https://github.com/braintree/braintree_android/blob/3.x/Braintree/src/main/java/com/braintreepayments/api/models/GooglePaymentConfiguration.java#L75). I remember removing it from the Core SDK since it would require a runtime dependency on the Google Pay SDK, but we probably kept it in when we went to GA because we needed to get a release out quickly.